apis/nucleus/src/components/listbox/hooks/selections/useSelectionsInteractions.js

```diff
@@ -16,6 +23,25 @@ export default function useSelectionsInteractions({ selectionState, selections,
     touchRangeSmall: false,
   });
 
+  useEffect(() => {
+    if (!loaderRef.current?._listRef?._outerRef) {
+      return undefined;
+    }
+    const preventGestureStart = (e) => e.preventDefault();
+    const preventGestureChange = (e) => e.preventDefault();
+    const preventGestureEnd = (e) => e.preventDefault();
+
+    const listRef = loaderRef.current._listRef._outerRef;
+    listRef.addEventListener('gesturestart', preventGestureStart);
+    listRef.addEventListener('gesturechange', preventGestureChange);
+    listRef.addEventListener('gestureend', preventGestureEnd);
+    return () => {
+      listRef.removeEventListener('gesturestart', preventGestureStart);
+      listRef.removeEventListener('gesturechange', preventGestureChange);
+      listRef.removeEventListener('gestureend', preventGestureEnd);
+    };
+  }, [loaderRef.current?._listRef?._outerRef]);
```

Comment:
this can maybe be a problem as 
loaderRef.current?._listRef?._outerRef could probably without updating the effect
if the update do not also trigger a rerender of the listbox